akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -834,8 +849,11 @@ class ClusterSingletonManager(singletonProps: Props, terminationMessage: Any, se
 
   @InternalStableApi
   def gotoOldest(): State = {
+    logInfo(
+      LogMarker("cluster.singleton.started"),
```

Comment:
rather than encoding the event type with a prefix, perhaps we could do something like cinnamon where we have an event type field? https://github.com/lightbend/cinnamon/blob/master/cinnamon-slf4j-events/src/main/java/cinnamon/slf4j/event/Slf4jEvents.java#L84

So that field can be indexed in elastic search

Reply:
The marker name is included as `tags` in Elasticsearch/Kibana so I thought it would be better to make use of that than adding another event type field. `tags` are indexed.

Not sure what you mean with the prefixing part. Each event type (marker name) should be unique, so we need some kind of naming convention. Maybe we should even prefix them with `akka`?

We could use non-dot naming if that is preferred. This example would anyway be equally long: `akkaClusterSingletonStarted`, but some others might be alright with shorter names.

Reply:
I think I'm leaning towards using non-dot naming for the marker names and prefix them with `akka`. More consistent with the naming of other mdc fields.